libs/core/src/components/pds-box/pds-box.tsx

```diff
@@ -163,7 +163,7 @@ export class PdsBox {
     ${this.border ? 'pds-box--border' : ''}
     ${this.borderRadius !== undefined && this.borderRadius.trim() !== '' ? `pds-border-radius-${this.borderRadius}` : ''}
     ${this.direction !== undefined && this.direction.trim() !== '' ? `pds-box-direction-${this.direction}` : ''}
-    ${this.display !== undefined && this.display.trim() !== '' ? `pds-box--display-${this.display}` : ''}
+    ${this.display !== undefined && this.display.trim() !== '' ? `pds---display-${this.display}` : ''}
```

Comment:
We're missing "box" here.

Reply:
Ah looks like that was addressed right when I commented. This can be ignored. LGTM!

Reply:
Great catch! Updated